packages/kn-plugin-workflow/pkg/command/deploy.go

```diff
@@ -98,14 +105,12 @@ func runDeploy(cmd *cobra.Command, args []string) error {
 			deploy,
 			"deploy",
 		); err != nil {
-			return err
+			return true, err
```

Comment:
Here it would be better to return value of the `isKnativeEventingBindingsCreated` property rather then specific value. Also the `isKnativeEventingBindingsCreated` is set to true on line 110, so I'm not sure about this return. 
```suggestion
			return isKnativeEventingBindingsCreated, err
```

Maybe it would be also good to add to the error handling also Println with failed deploy of kogito.yaml file

Reply:
Thank you for the review. You are right, it should return `isKnativeEventingBindingsCreated`. I accepted your suggestion and added `Println` message in case of error.